src/pages/wallet-overview/wallet-overview.spec.ts

```diff
@@ -263,4 +293,131 @@ describe("WalletOverviewPage", () => {
       });
     });
   })
+
+  it("should call initialize when the view is loaded", () => {
+    spyOn(comp, "initialize");
+
+    comp.ionViewDidLoad();
+
+    expect(comp.initialize).toHaveBeenCalled();
+  });
+
+  it("should have the funds visiblity to shown after they were invisible", () => {
+    comp.walletFundsVisibility = "hidden";
+
+    comp.fundsSwitch();
+
+    expect(comp.walletFundsVisibility).toBe("shown");
+    expect(comp.walletFundsVisibilityTransferButton).toBe("hidden");
+  });
+
+  it("should open the transfer page correctly", () => {
+    spyOn(navController, "push");
+
+    comp.openTransferPage();
+
+    expect(navController.push).toHaveBeenCalledWith(TransferPage);
+  })
+
+  it("should call balance and history after refreshing the wallet", () => {
+    spyOn(comp, "getWalletBalance");
+    spyOn(comp, "getTransactionHistory");
+
+    comp.currentWallet = <IWallet>{};
+    comp.currentWallet.publicKey = "";
+
+    comp.refreshWalletBalance();
+
+    expect(comp.getWalletBalance).toHaveBeenCalled();
+    expect(comp.getTransactionHistory).toHaveBeenCalled();
+  })
+
+  it("should not open landing page and show a toast after deleting a wallet while some wallets still exist", (done) => {
+    // Get all 3 wallets
+    comp.getAllWallets().then(data => {
+      // Delete first one
+      comp.deleteSelectedWallet(comp.currentWallet);
+
+      spyOn(comp, "openLandingPage");
+      spyOn(comp, "showToastMessage");
+
+      // Delete last one
+      comp.deleteSelectedWallet(comp.currentWallet);
+
+      expect(comp.openLandingPage).not.toHaveBeenCalled();
+      expect(comp.showToastMessage).not.toHaveBeenCalled();
+      
+      done();
+    });
+  })
+
+  it("should return 7 fixed numbers if the picked currency is not in the if condition", () => {
```

Comment:
The description of this test is a bit unclear. Which 'if condition' are you referring to? Also I am assuming 7 fixed numbers refers to the amount of decimals?